tools/flaky-test-reporter/README.md

```diff
@@ -14,18 +14,46 @@ Flags for this tool are:
   Github API calls.
 - `--slack-account` specifies the path of file containing Slack token for Slack
   web API calls.
+- `skip-report` skips all Github/Slack activities. This is used for the
+  purpose of data collection.
 - `--dry-run` enables dry-run mode.
 
 ### IMPORTANT: This tool is _NOT_ intended to run locally, as this could interfere with real Github issues and potentially flood Knative Slack channels
 
+## How To Debug/Verify Changes
+
 For debugging purpose it's highly recommended to start with `--dry-run` flag, by
-passing this flag it will only collect information from GCS/Github/Slack, and
+passing this flag it will only collect information from GCS/Github, and
 all the manipulations of Github/Slack resources are omitted.
 
+### Requirement
+
+- `GCP token`: Create a service account by visiting
```

Comment:
Link to https://cloud.google.com/iam/docs/creating-managing-service-account-keys instead of explaining it here

Reply:
done